### PR TITLE
Consolidate Newtonsoft versions

### PR DESCRIPTION
**Pull request by am11**
```
Newtonsoft.Json => v13.0.1
Newtonsoft.Json.Bson => v1.0.2
```

### CODE REVIEW

**Review by jkoritzinsky — COMMENTED** [review]
We can't move the projects under the installer test assets to use the versions in eng/Versions.props as they are copied over to Helix and built independently of the rest of the repo.

**Comment by am11** [comment]
@jkoritzinsky, I know. That's why I have added an additional property for those projects.

**Comment by jkoritzinsky** [comment]
I totally missed that in the diff. This looks good to me then! I'd like to get some feedback from the owners of the host tests before approving. @vitek-karas @agocke

File: src/installer/tests/Assets/TestUtils/TestProjects.props [hunk 1]
```diff
@@ -13,5 +13,7 @@
       win-x86 tests that assumed a win-x64 app host RID based on the runner SDK.
     -->
     <AppHostRuntimeIdentifier>$(TestTargetRid)</AppHostRuntimeIdentifier>
+
+    <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
```
Comment (vitek-karas) [inline src/installer/tests/Assets/TestUtils/TestProjects.props:17]: Is this necessary? I would have thought that this gets propagated from `Versions.props`...

**Review by vitek-karas — APPROVED** [review]
The host/hostests changes look good. Thanks a lot!

File: src/installer/tests/Assets/TestUtils/TestProjects.props (hunk 1 above)
Comment (am11) [reply src/installer/tests/Assets/TestUtils/TestProjects.props:17]: This is used by host test projects, which are published during the test runs. At that point artifices/tests is copied to remote machine (helix) with TestUtils, with no access to `Versions.props`. https://github.com/dotnet/runtime/blob/3a1fd4d2e5cbb4850ae608a5867c79c60781da25/src/installer/tests/Assets/TestUtils/TestProjects.props#L2-L5

Comment (vitek-karas) [reply src/installer/tests/Assets/TestUtils/TestProjects.props:17]: Thanks a lot for the explanation!

**Pull request merged**